datalad/resources/website/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Datalad Repository</title>
+        <link rel="shortcut icon" type="image/x-icon" href="http://datalad.org/pics/favicon-16x16.png" />
+	<link rel="stylesheet" type="text/css" href="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.10.5/css/jquery.dataTables.css">
+        <style type="text/css" class="init">
+          body{
+           max-width: 1200px;
+           margin: 2em auto;
+         }
+	 td.dir {font-weight: bold; color: #00A1CB; }
+	 td.git {font-weight: bold; color: #00BA88; }
+	 td.annex {font-weight: bold; color: #EC6C20; }
+	 td.link-broken {color:#E53B51; }
+	 td.link {color: #FFD03C;  }
+	 td.file {color: #67CDDC;  }
+	</style>
+    </head>
+    <body>
+        <h1><img src='http://datalad.org/pics/favicon.svg' width="24">Datalad Repository</h1>
+	<table id="directory" class="display" width="100%" cellspacing="0"></table>
+	<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.12.3.min.js"></script>
+	<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.10.5/jquery.dataTables.min.js"></script>
+	<script type="text/javascript" charset="utf8" src="https://raw.githubusercontent.com/blueimp/JavaScript-MD5/master/js/md5.min.js"></script>
+	<script>
+	 function directory() {
+	     // construct path to metadata json based on directory to be rendered argument passed in url
+	     var metadata_dir = '.git/datalad/metadata/'
+	     var metadata_path = getParameterByName('dir') ? getParameterByName('dir').slice(1,-1) : '/';
+	     var metadata_url = window.location.pathname + metadata_dir + md5(metadata_path);
+	     var parent = false;
+
+	     var table = $('#directory').DataTable( {
+		 "async": true,    //async get json
+		 "paging": false,  //ensure scrolling instead of pages
+		 "ajax": {         //specify url to get json from ajax
+		     "url": metadata_url,
+		     "dataSrc": "nodes"
+		 },
+	         "order": [[ 6, "desc" ], [ 0, 'asc' ]],
+		 "columns": [      //select columns and their names from json
+		     { data: "name", title: "Name" },
+                     { data: "date", title: "Last Modified", className: "dt-center", width: "20%" },
+                     { data: "size", title: "Size", className: "dt-center" },
```

Comment:
my emacs shows them w/o the varying indentation.
seems like more of a tabs vs spaces editor rendering issue :/
made it all spaces for now. hopefully that fixes it.